### PR TITLE
Update foreman package dependencies

### PR DESCRIPTION
**Pull request by ekohl**
Replaces https://github.com/theforeman/foreman-packaging/pull/5268 which missed a few.

### CODE REVIEW

File: packages/foreman/foreman/foreman.spec [hunk 1]
```diff
@@ -259,7 +263,11 @@ BuildRequires: systemd
 # start package.json devDependencies BuildRequires
 BuildRequires: %{?scl_prefix}npm(@babel/core) >= 7.7.0
 BuildRequires: %{?scl_prefix}npm(@babel/core) < 8.0.0
-BuildRequires: %{?scl_prefix}npm(@theforeman/builder) >= 4.0.7
+BuildRequires: %{?scl_prefix}npm(@babel/plugin-transform-modules-amd) = 7.9.0
+BuildRequires: %{?scl_prefix}npm(@babel/plugin-transform-modules-commonjs) = 7.9.0
+BuildRequires: %{?scl_prefix}npm(@babel/plugin-transform-modules-systemjs) = 7.9.0
+BuildRequires: %{?scl_prefix}npm(@babel/preset-env) = 7.9.5
```
Comment (ekohl) [inline packages/foreman/foreman/foreman.spec:269]: I think these are not needed for us. I'll remove it for now

Comment (tbrisker) [inline packages/foreman/foreman/foreman.spec:269]: These aren't really required, they are provided by foreman-js and in the package-json just to pin the version used by npm

**Comment by ekohl** [comment]
Looks like `amazing_print` hasn't been mashed yet so repoclosure fails.

**Comment by tbrisker** [comment]
[test rpm]

**Comment by tbrisker** [comment]
:green_apple: now

**Pull request merged**